### PR TITLE
remove bentheelder from oncall

### PR DESCRIPTION
**Pull request by BenTheElder**
- I am not in the oncall rotation anymore (cl/238735365) and will be focusing efforts on [kubernetes/kubernetes](https://github.com/kubernetes/kubernetes), [kubernetes-sigs/kind](https://github.com/kubernetes-sigs/kind), and GKE. 
  - cleanup OWNERS
  - cleanup SECURITY_CONTACTS
  - cleanup testgrid alerts
  - cleanup TODO
- noticed experiment/bootstrap on the way, that is all (my) dead code, removed it.

/assign @spiffxp

### CODE REVIEW

**Comment by BenTheElder** [comment]
NOTE: `SECURITY_CONTACTS` needs new people, as @rmmh is not with the project.

File: SECURITY_CONTACTS [hunk 1]
```diff
@@ -10,6 +10,5 @@
 # DO NOT REPORT SECURITY VULNERABILITIES DIRECTLY TO THESE NAMES, FOLLOW THE
 # INSTRUCTIONS AT https://kubernetes.io/security/
 
-bentheelder
 krzyzacy
 rmmh
```
Comment (krzyzacy) [inline SECURITY_CONTACTS]: (maybe put rest of oncalls here and remove Ryan? I'm not sure if I want to solo here :joy: )

File: images/OWNERS [hunk 2]
```diff
@@ -1,7 +1,6 @@
 # See the OWNERS docs at https://go.k8s.io/owners
 
 approvers:
-- bentheelder
```
Comment (spiffxp) [inline images/OWNERS]: I volunteer as tribute (I leave it to the other two approvers to decide if I should come in as a reviewer first)

File: prow/plugins/welcome/welcome_test.go [hunk 3]
```diff
@@ -32,7 +32,7 @@ import (
 	"k8s.io/test-infra/prow/plugins"
 )
 
-// TODO(bentheelder): these tests are a bit lame.
+// TODO: these tests are a bit lame.
```
Comment (spiffxp) [inline prow/plugins/welcome/welcome_test.go]: I feel like you should go ahead and drop this TODO, it's too subjective

File: testgrid/config.yaml [hunk 4]
```diff
@@ -3641,37 +3641,37 @@ dashboards:
   - name: GCE, v1.13 (release)
     description: Runs conformance tests using kubetest against kubernetes release 1.13 stable tag on GCE
     test_group_name: ci-kubernetes-gce-conformance-stable-1-13
-    # TODO(bentheelder): there's probably a more appropriate alias to alert this to
+    # TODO: there's probably a more appropriate alias to alert this to
```
Comment (spiffxp) [inline testgrid/config.yaml]: I volunteer as tribute for these testgrid TODO's

File: OWNERS [hunk 5]
```diff
@@ -2,7 +2,6 @@
 
 approvers:
 - amwat # oncall
-- BenTheElder # oncall
```
Comment (spiffxp) [inline OWNERS]: `# - BenTheEnderl # emeritus`?

File: config/OWNERS [hunk 6]
```diff
@@ -1,8 +1,7 @@
 # See the OWNERS docs at https://go.k8s.io/owners
 
 approvers:
-- bentheelder
```
Comment (spiffxp) [inline config/OWNERS]: I volunteer as tribute

File: images/OWNERS (hunk 2 above)
Comment (BenTheElder) [reply images/OWNERS]: @ixdy is probably relatively done with images/, @krzyzacy WDYT?

Comment (krzyzacy) [reply images/OWNERS]: yep, thanks @spiffxp

File: SECURITY_CONTACTS (hunk 1 above)
Comment (BenTheElder) [reply SECURITY_CONTACTS]: dropped ryan and added an @spiffxp todo to add more

File: config/OWNERS (hunk 6 above)
Comment (BenTheElder) [reply config/OWNERS]: done

File: images/OWNERS (hunk 2 above)
Comment (BenTheElder) [reply images/OWNERS]: done

File: testgrid/config.yaml (hunk 4 above)
Comment (BenTheElder) [reply testgrid/config.yaml]: done

File: prow/plugins/welcome/welcome_test.go (hunk 3 above)
Comment (BenTheElder) [reply prow/plugins/welcome/welcome_test.go]: done

File: OWNERS (hunk 5 above)
Comment (BenTheElder) [reply OWNERS]: https://github.com/kubernetes/test-infra/issues/10747 ?

**Comment by spiffxp** [comment]
nit: drop the "per uri-to-pull-request" in `b77c2cb`'s commit message, maybe "drop subjective TODO" instead

File: OWNERS (hunk 5 above)
Comment (spiffxp) [reply OWNERS]: all I care about here is giving you some recognition as emeritus, this is the laziest way for now (we do this for leads in sigs.yaml via another field)

Comment (BenTheElder) [reply OWNERS]: got it, my brain missed the `#`, this is done.

**Comment by BenTheElder** [comment]
> nit: drop the "per uri-to-pull-request" in b77c2cb's commit message, maybe "drop subjective TODO" instead

done

**Pull request merged**